redwoods-insurance-salesforce/package.json

```diff
@@ -35,7 +35,8 @@
         "lint-staged": "^10.2.11",
         "prettier": "^2.0.5",
         "prettier-plugin-apex": "^1.5.0",
-        "semver": "^7.3.2"
+        "semver": "^7.3.2",
+        "jest": "25.5.4"
```

Comment:
This is not needed.

Reply:
Jest is needed. Without the debugger won't work. While our sfdx-jest-lwc plugin installs Jest, it's not creating the default node_modules/.bin/jest symlink which means the debugger will not work. Even though a launch.json can specify using a particular jest path, the debugger does not honor it. I went 9 rounds with this yesterday, and it took a 45 minute call with another developer ev to figure out how to solve it. I have no idea why *our* package isn't making the symlink, but it's not. That is outside the scope of this PR. leaving jest as a dependency fixes the issue, and is harmless at best.

Reply:
steps to reproduce:
1. clone the repo
2. run npm install
3. check for the presence of node_modules/.bin/jest it's not there.

Reply:
@codefriar I must agree with @muenzpraeger on the fact that overriding Jest version in our project is not harmless. This could cause issues and we do not want to maintain this on the long run.

Having said that, I do confirm that there's no `node_modules/.bin/jest` after installing the project (curiously `sfdx-lwc-jest --debug` command doesn't fail though).
Can you create an issue with all details in the [sfdx-lwc-jest repo](https://github.com/salesforce/sfdx-lwc-jest)?

Reply:
@pozil Thanks Philippe, I'm opening an issue on that repo now, and will link it here when I'm done. in the meantime, how do you want to solve this? Without Jest here, I cannot use the debugger, and without the debugger, writing jest tests is a particularly evil form of punishment ;) I should note that I have also tried a -g install of jest and using the launch.json file to hard code it to /usr/local/bin/jest but this also fails.

Reply:
https://github.com/salesforce/sfdx-lwc-jest/issues/155 has been filed.

Reply:
Cool, thanks Kevin.